### DISCUSSION TITLE
Submission error: 'NoneType' object has no attribute 'forecast'.

### DISCUSSION THREAD

**Discussion by shaido987** [PaddlePaddle/PaddleSpatial#106] [BaiduKDDCup2022] [answered]
Hello, 

could I get some more details on what went wrong with my submission? The given error is: 'NoneType' object has no attribute 'forecast', so I suspect that there is an issue with one of the imports.

Team name: didadida_hualahuala
Environment: PyTorch
Time: 2022-05-26 11:13 (any of the ones 2022-05-26 should be relevant)

**Comment by zhanshijinwat** [comment]
I also have the same probelm！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！

**Comment by ramber1836** [comment]
We are working on it.
Thanks.

Regards,
Xinjiang Lu

**Comment by ramber1836** [comment, marked as answer]
More tractable error msg is available.

**Reply by shaido987** [reply]
Thanks for the update.
